### ISSUE TITLE
Support gen 1 of the Powerview Hub?

### ISSUE THREAD

**Issue by EdSteenhoek** [HUSSBV/homey_nl.luxaflex.powerview#3]
It seems that the 1st gen Powerview hub is not supported. Can that be added?

I have installed the app on my Homey and once the IP address has been entered and the devices is turned on in Homey, it does see the scenes but they are not executed...
My assumption is that this is not a bug but that only newer hubs are supported even though there is no word on the app page about such a limitation.

**Comment by rogierhofboer** [comment]
The beta version in the Athom app store did not work with my v1 hub. The current github version is already working without any problems, not sure why an updated beta is not released yet on the Athom store, I am not from HUSS / Luxaflex / Athom..

In case you need help on installing a version from github source codd, please let me known.

**Comment by EdSteenhoek** [comment]
Thanks! Will give it a try this weekend.

I think I can work it out but thanks for the offer. If I don't succeed, I'll reach out.

**Comment by rogierhofboer** [comment]
Did you manage to try? Does it work?

**Comment by EdSteenhoek** [comment]
Did install the latest version from github. That didn't give a problem.
But didn't have time to solve the next issue: the scene that the hub nows are executed but the Luxaflex blinds don't move. Not even when using the Powerview app. No clue yet why because it used to work from the Powerview app. Hope to find time this weekend to further investigate

**Comment by EdSteenhoek** [comment]
Fixed most issues today.

First, I identified that the Powerview app mistakenly thought that the batteries were empty. All are connected with a power adapter and don't use batteries... Refreshed the status in the Powerview app and 2 out of 3 started working again. The 3rd worked also after I disconnected and reconnected the power.
Second, the scenes appeared to be OK but when I went into the scene config and changed the position of the blinds just a fraction and the hit the button 'use current position' (or whatever it is called in English...) only then did they become responsive. No clue why, but this seems to have fixed it.

What remains is that a second scene command isn't executed from a flow in Homey. This has to be a timing issue because when I use the Powerview App, executing 2 scenes immediately after one and another doesn't work either. Yet, what I see in the Homey flow is, that the 2nd command isn't even executed. If I test the flow, the first command gets the green checkmark but the second command doesn't. It even doesn't start spinning (meaning it is trying to execute). I tried with a 10 seconds delay but that didn't resolve the issue. 

The reason why I have 2 commands in a row is that I'm not convinced that the blinds always respond. A typical flow that is triggered when I want to almost close the open blinds is this: 1) close the blinds completely and 2) open them just a little.  May not be needed, but in case either one fails, the end-result is still giving the effect that I want: keeping the sun out.
